src/main/java/org/apache/bcel/classfile/LocalVariableTypeTable.java

```diff
@@ -67,8 +67,8 @@ public LocalVariableTypeTable(final int name_index, final int length, final Loca
         this.local_variable_type_table = local_variable_table;
     }
 
-    LocalVariableTypeTable(final int nameIdx, final int len, final DataInput input, final ConstantPool cpool) throws IOException {
-        this(nameIdx, len, (LocalVariable[]) null, cpool);
+    LocalVariableTypeTable(final int name_index, final int length, final DataInput input, final ConstantPool cpool) throws IOException {
+        this(name_index, length, (LocalVariable[]) null, cpool);
```

Comment:
I am -1 to using underscores in Java. If anything,  we should change existing underscores to camel case. I like changing idx to index.

Reply:
Although I'm not insisting, *_index is a pretty common naming scheme throughout the project (about 600 instances). Actually *idx is common too (about 200 instances), but the other constructor in this class already uses name_index as an argument, so I thought this change fits more naturally than the other way around.

Reply:
Unless you plan on following up with changing `nameIdx` to `nameIndex`, we should close this PR. Please advise.